.github/workflows/programequity_slack.yml

```
@@ -0,0 +1,24 @@
+on:
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created, edited]
+  pull_request:
```

We should realistically use [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) here instead of `pull_request` to ensure it works for PRs from forks as well.

```suggestion
  pull_request_target:
```